firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/impl/AabUpdaterTest.java

```diff
@@ -68,6 +67,7 @@ public class AabUpdaterTest {
 
   private AabUpdater aabUpdater;
   private ShadowActivity shadowActivity;
+  private static ExecutorService testExecutor;
```

Comment:
Shouldn't be static since it's initialized in a `@Before` method. But I'm curious why you changed this from the "private static final" that was used before.

Reply:
Ah, true, done.

I changed it to re-initialize in the `@Before` because I noticed that only having one static executor was causing state to bleed between tests.